common/app/views/fragments/containers/facia_cards/standardHeaderMeta.scala.html

```diff
@@ -19,7 +19,7 @@
                     "tone-comment" -> isContributor,
                     "tone-colour" -> isContributor))">
                     @href.map { href =>
-                    <a data-link-name="section heading" href="@LinkTo {/@href}">
+                    <a data-link-name="section heading" href="/@LinkTo(href, None)">
```

Comment:
Have you checked what else uses this template? Does anything other than onward use it?